bindings/netstandard/ElectionGuard/ElectionGuard.Decryption.Tests/ElectionRecord/TestElectionRecord.cs

```diff
@@ -0,0 +1,89 @@
+
+using ElectionGuard.Decryption.Decryption;
+using ElectionGuard.Decryption.ElectionRecord;
+using ElectionGuard.Decryption.Tests.Decryption;
+using ElectionGuard.ElectionSetup.Tests.Generators;
+using ElectionGuard.Encryption.Utils.Generators;
+
+namespace ElectionGuard.Decryption.Tests.ElectionRecord;
+
+[TestFixture]
+public class TestElectionRecord : DisposableBase
+{
+    private const int BALLOT_COUNT_CAST = 10;
+    private const int BALLOT_COUNT_CHALLENGED = 2;
+    private const int BALLOT_COUNT_SPOILED = 2;
+    private const int NUMBER_OF_GUARDIANS = 5;
+    private const int QUORUM = 3;
+
+    private TestDecryptionData Data = default!;
+
+    private DecryptionMediator Mediator = default!;
+
+    private DecryptionResult Result = default!;
+
+    [OneTimeSetUp]
+    public void OneTimeSetup()
+    {
+        // RunSetup();
```

Comment:
Since the setup call moved into the test call itself, this could be deleted instead of just commented out.